lib/utils/device/deploy.ts

```diff
@@ -74,7 +74,7 @@ async function environmentFromInput(
 	serviceNames: string[],
 	logger: Logger,
 ): Promise<ParsedEnvironment> {
-	const { exitWithExpectedError } = await import('../patterns');
+	const { exitWithExpectedError } = await import('../../errors');
```

Comment:
The `errors` module is already imported at the top of the file, so maybe `exitWithExpectedError` could be included there too. (3 occurrences in this file)

Reply:
Looks like that's a different errors module (`utils/device/errors.ts`).